terraform/fec_rds.tf

```diff
@@ -110,6 +110,11 @@ resource "aws_db_parameter_group" "fec_default" {
         value = "ddl"
     }
 
+    parameter {
+        name  = "log_min_duration_statement"
+        value = "1000" # in ms - 1s
+    }
+
```

Comment:
Do we want to keep this at `1,000` for our default and adjust to 1ms manually when we are actively profiling for a complete picture or just set it to 1ms in general, based on some of the other feedback we had received?

Reply:
IMO, the former - set it to `1000` for our default, and override it when we are actively profiling.

Reply:
👍  I like the idea of keeping this as a default and overriding during periods when we need more data.

Reply:
Sounds good!  Let's coordinate a plan for deployment.  This shouldn't result in any restarts but just want to be sure and monitor just in case. :-)

Reply:
I just confirmed that this is a dynamic setting in [RDS' PostgreSQL parameter group documentation](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.PostgreSQL.CommonDBATasks.html#Appendix.PostgreSQL.CommonDBATasks.Parameters) (you'll have to search for the `log_min_duration_statement` setting in the table), so there should be zero downtime and no restarts.